R/tidyeval-across.R

```diff
@@ -60,7 +60,10 @@ across_funs <- function(funs, env, data, dots, names_spec, fn, evaluated = FALSE
     funs <- eval(funs, env)
     return(across_funs(funs, env, data = data, dots = dots, names_spec = NULL, fn = fn, evaluated = TRUE))
   } else {
-    cli_abort("{.arg .fns} argument to {.fun dbplyr::across} must be a NULL, a function, formula, or list")
+    cli_abort(
+      "{.arg .fns} argument to {.fun dbplyr::across} must be a NULL, a function, formula, or list",
+      call = NULL
```

Comment:
For consistency, you could maybe try call = `quote(dbplyr::across())` and then remove from the message

Reply:
> then remove from the message

I'm not quite sure how I could remove it.

Reply:
I just meant that you no longer need `to {.fun dbplyr::across}` in the message.